docs/starting_systemd.md

```diff
@@ -0,0 +1,121 @@
+# Starting OVOS - systemd
+OVOS being a modular system has several pieces that can start individually.
+
+user systemd service files are the recomended way to start each module, but may be started in other ways also.
```

Comment:
```suggestion
User `systemd` service files are the recommended way to start each module, but may be started in other ways also.
```

Reply:
Consider striking "but may be started in other ways also" or adding a couple of other ways they could be started.

Reply:
Might check this one too.